lua/orgmode/org/mappings.lua

```diff
@@ -258,6 +258,58 @@ function OrgMappings:change_date()
   Calendar.new({ callback = cb, date = date }).open()
 end
 
+function OrgMappings:set_priority()
+  local item = Files.get_closest_headline()
+  local todo = item.todo_keyword
+  local high_priority = config.org_priority_highest
+  local low_priority = config.org_priority_lowest
+  local is_alphabetical = string.find(high_priority, '%a')
+
+  local start = high_priority
+  local fin = low_priority
+  if is_alphabetical then
+    start = string.byte(high_priority)
+    fin = string.byte(low_priority)
+  end
+
+  local enumerated = {}
+  for i = start, fin do
+    local key = i
+    if is_alphabetical then
+      key = string.char(i)
+    end
+
+    table.insert(enumerated, { choice_value = key, choice_text = key, desc_text = '', desc_hl = 'Comment' })
+  end
+
+  table.insert(enumerated, { choice_value = ' ', choice_text = '<Space>', desc_text = 'Remove', desc_hl = 'Comment' })
+
+  local choice = utils.choose(enumerated)
+  local linenr = item.range.start_line
```

Comment:
I would like to move all this logic of setting priority to the `Section` file as `set_priority` method, that just accepts the priority valueas an argument.